eng/testing/runsettings.targets

```diff
@@ -46,6 +47,12 @@
                       WriteOnlyWhenDifferent="true"
                       Overwrite="true" />
     
+    <WriteLinesToFile File="$(VsCodeRunSettingsOutputFilePath)"
```

Comment:
Why do we need the same file in 2 places? Why won't one place work?

Reply:
The problem is that the other places contain build configuration in the paths `TargetOS-Configuration-Arch`, and that changes (not on codespaces) but it will locally on different dev boxes if you build for a different config or arch, so we need the VS Code to be on the same location to be able to specify the path to it on the vscode settings file.

Reply:
I don't understand why Codespaces isn't treated like any other "local dev box" here. What about it makes it different?

> we need the VS Code to be on the same location to be able to specify the path to it on the vscode settings file.

Why isn't this a problem when I use VS Code locally?

Reply:
I think my reply wasn't clear enough.

I'm actually treating Codespaces and local dev box as the same. What I was saying is that for Codespaces we do know what the build configuration is (because we prebuild it), but on local dev box we don't. 

That is why we can't use the already generated .runsettings file because its path is configuration specific and since the only way to point to OmniSharp the `.runsettings` file is via a global setting statically we need a relative path within the repo that is the same on all machines and that doesn't change based on build configurations, so that's why we can't use the already generated file.

Reply:
ah, got it. That makes sense.